docs/installation.rst

```diff
@@ -14,15 +14,19 @@ Configuration
 ---------------
 
 
-Add ``djstripe`` to your ``INSTALLED_APPS``:
+Add ``djstripe`` to your ``INSTALLED_APPS``. You will also need the `sites` framework enabled.:
```

Comment:
Looks like the `Site` model is still being used here in 1.0.0 - https://github.com/kavdev/dj-stripe/blob/1.0.0/djstripe/models.py#L95